pages/typeform-101.vue

```diff
@@ -0,0 +1,412 @@
+<template>
+  <div>
+    <div class="feature_container">
+      <div class="feature__hero">
+        <div class="container">
+          <div
+            class="d-flex flex-column justify-content-center align-items-center text-center mt-5"
+          >
+            <h1 class="hero__heading pt-3">
+              Designing Forms with <span class="hglt">Visual Brilliance!</span>
+            </h1>
+            <p class="hero__subheading mt-3">
+              Unlock the true potential of form building with Typeform. At
+              Formester, we're committed to providing you with a comprehensive
+              guide to help you navigate the world of form creation. Discover
+              the features, tips, and tricks that make Typeform a top choice for
+              form builders!
+            </p>
+          </div>
+          <div class="d-flex align-items-center justify-content-center mt-5">
+            <nuxt-img
+              src="/typeform-101/typeform-hero.svg"
+              alt="Hero-Image"
+              class="img-fluid hero__image"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <TypeformKeyFeatures :feature-data="typeformKeyFeatures" />
+    <StandOutFeatureSection :featureData="typeformStandOutFeatures" />
+    <PopularUseCase :featureData="typeformPopularUseCases" />
+    <Limitations :featureData="typeformLimitations" />
+    <FormBuilderComparisionTable
+      heading="Comparing Jotform with Other Leading Form Builders"
+    />
+    <WhyChooseFormester />
+    <FAQwithCategories :categories="typeformFaqs" />
+    <CallToActionSection
+      :content="{
+        heading: 'Start Your Formester Journey Today',
+        subheading:
+          'Start with a Free Sign up and experience the power of Formester firsthand',
+      }"
+    />
+  </div>
+</template>
+
+<script>
+import TypeformKeyFeatures from '../components/competitors/KeyFeatures.vue'
```

Comment:
Is there a way to access using an absolute path? something like `@`

Reply:
the path was automatically filled after entering the component name. 🥲